e2e/content-services/document-list/document-list-component.e2e.ts

```diff
@@ -268,39 +275,27 @@ describe('Document List Component', () => {
         });
 
         it('[C260112] Should be able to sort by name (Ascending)', () => {
-            loginPage.loginToContentServicesUsingUserModel(acsUser);
-            contentServicesPage.goToDocumentList();
-            contentServicesPage.sortAndCheckListIsOrderedByName('asc');
+            expect(contentServicesPage.sortAndCheckListIsOrderedByName('asc')).toBe(true, 'List is not sorted.');
         });
 
         it('[C272770] Should be able to sort by name (Descending)', () => {
-            loginPage.loginToContentServicesUsingUserModel(acsUser);
-            contentServicesPage.goToDocumentList();
-            contentServicesPage.sortAndCheckListIsOrderedByName('desc');
```

Comment:
have we moved this step to somewhere? it's deleted here, but I don't see it moved.

Reply:
Yes they were moved to the beforeAll